xml/System.Net.PeerToPeer.Collaboration/PeerContact.xml

```diff
@@ -327,7 +327,7 @@
       <Docs>
         <param name="obj">Compares the <see cref="T:System.Net.PeerToPeer.Collaboration.PeerContact" /> instance to the specified object.</param>
         <summary>Compares the <see cref="T:System.Net.PeerToPeer.Collaboration.PeerContact" /> instance to the specified object.</summary>
-        <returns>Returns <see langword="true" /> if the <see cref="T:System.Net.PeerToPeer.Collaboration.PeerContact" /> instance provided has matching data, else <see langword="false" />. This method also returns <see langword="false" /> if the passed parameter is <see langword="null" />.</returns>
+        <returns><see langword="true" /> if the <see cref="T:System.Net.PeerToPeer.Collaboration.PeerContact" /> instance provided has matching data, else <see langword="false" />. This method also returns <see langword="false" /> if the passed parameter is <see langword="null" />.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if the <see cref="T:System.Net.PeerToPeer.Collaboration.PeerContact" /> instance provided has matching data; otherwise, <see langword="false" />. This method also returns <see langword="false" /> if the passed parameter is <see langword="null" />.</returns>
```